app/javascript/components/Greetings.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from "react";
```

Comment:
- [ ] It would be better if we set up Redux and create the store, actions and reducers for our application :fireworks: